datahub-web-react/src/app/ingest/source/builder/RecipeForm/dbt_cloud.tsx

```diff
@@ -0,0 +1,296 @@
+import React from 'react';
+import styled from 'styled-components';
+import { get } from 'lodash';
+import { RecipeField, FieldType, setFieldValueOnRecipe } from './common';
+
+const TipSection = styled.div`
+    margin-bottom: 12px;
+`;
+
+export const DBT_CLOUD = 'dbt-cloud';
+
+export const DBT_CLOUD_TOKEN: RecipeField = {
+    name: 'token',
+    label: 'API Token',
+    tooltip:
+        'A service account API token for extracting metadata from dbt Cloud APIs. This token must have the privileges required to read metadata (Metadata permission set).',
```

Comment:
This line is a little unclear to me. What does "Metadata permission set" mean here?

This token must have the privileges required to read metadata (Metadata permission set)

Reply:
Could also use some markdown to highlight the specific permission